api/src/test/java/org/openmrs/module/fhir2/providers/r3/EncounterFhirResourceProviderTest.java

```diff
@@ -333,4 +334,41 @@ public void deleteEncounter_shouldThrowResourceNotFoundExceptionWhenIdRefersToNo
 		when(encounterService.delete(WRONG_ENCOUNTER_UUID)).thenReturn(null);
 		resourceProvider.deleteEncounter(new IdType().setValue(WRONG_ENCOUNTER_UUID));
 	}
+	
+	@Test
+	public void searchForEncounters_shouldReturnEncounterEverything() {
```

Comment:
the method name is getEncounterEverything  not searchForEncounters,so we could use the method name

Reply:
sure will update it.